SimpleWAWS.Console/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System;
```

Comment:
Okay. I'll check that `PutInDesiredState` is called if a functions Container or storage account is absent on the resourcegroup.